app/api/v1/views.py

```diff
@@ -241,48 +234,34 @@ def more_info(suborder_number):
 
     if request.method == 'POST':
         order_info = SearchFunctions.format_first_result(search_queries(suborder_number=suborder_number,
-                                                                        search_type="print"))
+                                                                        search_type='print'))
 
         return jsonify(order_info=order_info), 200
 
 
-@api.route('/orders/<int:order_id>', methods=['GET'])
-@login_required
-def get_single_order(order_id):
-    """
-    :param order_id:
-    :return: the orders with that specific client id that was passed
-    """
-    orders = [order.serialize for order in Orders.query.filter_by(client_id=order_id).all()]
-
-    if len(orders) == 0:
-        abort(404)
-
-    return jsonify(orders=orders), 200
-
-
 @api.route('/tax_photo/<string:suborder_number>', methods=['GET', 'POST'])
 @login_required
 def tax_photo(suborder_number):
```

Comment:
Can you add a TODO comment here for a docstring?

Reply:
Added TODO.